apps/discord-bot/internal/service/service.go

```diff
@@ -63,89 +63,83 @@ func New() Service {
 	}
 }
 
-func (s *service) GetDiscordBotCommand(ctx context.Context, commandName string, discordServerId string) (*models.BotCommand, error) {
-	return s.DB.GetDiscordBotCommand(ctx, commandName, discordServerId)
+func (s *service) GetUserBotCommand(ctx context.Context, commandName string, discordServerId string) (*models.BotCommand, error) {
+	return s.DB.GetUserBotCommand(ctx, "discord", commandName, discordServerId)
 }
 
-func (s *service) CreateDiscordBotCommand(ctx context.Context, commandName string, commandContent string, discordServerId string, createdBy string) (*string, error) {
-	infoText, err := s.DB.CreateDiscordBotCommand(ctx, commandName, commandContent, discordServerId, createdBy)
+func (s *service) GetGlobalBotCommand(ctx context.Context, commandName string) (*models.BotCommand, error) {
+	return s.DB.GetGlobalBotCommand(ctx, commandName)
+}
+
+func (s *service) CreateCommand(ctx context.Context, commandName string, commandContent string, discordServerId string, createdBy string) (*string, error) {
+	infoText, err := s.DB.CreateBotCommand(ctx, "discord", commandName, commandContent, discordServerId, createdBy)
```

Comment:
Rather than directly using "discord" as a raw string throughout the code, it would be more maintainable and readable to define it as a constant. Perhaps we could name it PlatformDiscord or choose another descriptive name. This approach will reduce potential errors and make future changes more straightforward. Additionally, it's important to ensure that all other occurrences of the raw string "discord" in the code are updated to use this constant for consistency.